bin/adam-submit

```diff
@@ -57,17 +57,17 @@ ADAM_CLI_JAR=
 if [ -d "$SCRIPT_DIR/repo" ]; then
   ASSEMBLY_DIR="$SCRIPT_DIR/repo"
 else
-  ASSEMBLY_DIR="$SCRIPT_DIR/adam-cli/target"
+  ASSEMBLY_DIR="$SCRIPT_DIR/adam-assembly/target"
 fi
 
-num_jars="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam-cli_[0-9A-Za-z\.-]*\.jar$" | grep -v javadoc | wc -l)"
+num_jars="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam-assembly_[0-9A-Za-z\.-]*\.jar$" | grep -v javadoc | wc -l)"
 if [ "$num_jars" -eq "0" ]; then
   echo "Failed to find ADAM cli assembly in $ASSEMBLY_DIR." 1>&2
   echo "You need to build ADAM before running this program." 1>&2
   exit 1
 fi
 
-ASSEMBLY_JARS="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam-cli_[0-9A-Za-z\.-]*\.jar$" | grep -v javadoc || true)"
+ASSEMBLY_JARS="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam-assembly_[0-9A-Za-z\.-]*\.jar$" | grep -v javadoc || true)"
```

Comment:
I wonder if it might be preferable to use `<finalName>adam_2.10</finalName>` for the fat jar artifact?